tests/test_solution.py

```diff
@@ -19,30 +36,53 @@ def example_solution() -> Solution:
         transducer_id="trans_456",
         created_on=datetime(2024, 1, 1, 12, 0),
         description="This is a test solution for a unit test.",
-        delays=np.array([0.0, 1.0, 2.0, 3.0]),
-        apodizations=np.array([0.5, 0.75, 1.0, 0.85]),
+        delays=np.array([[0.0, 1.0, 2.0, 3.0]]),
+        apodizations=np.array([[0.5, 0.75, 1.0, 0.85]]),
         pulse=Pulse(frequency=42),
         sequence=Sequence(pulse_count=27),
-        focus=Point(id = "test_focus_point"),
-        target=Point(id = "test_target_point"),
-        simulation_result=xarray.Dataset(
-            {'pnp': (['x', 'y', 'z'], rng.random((3, 2, 3)))},
-            coords={'x': np.linspace(0, 1, 3), 'y': np.linspace(0, 1, 2), 'z': np.linspace(0, 1, 3)},
+        foci=[Point(id="test_focus_point")],
+        target=Point(id="test_target_point"),
+        simulation_result=xa.Dataset(
```

Comment:
Thanks. I do prefer having a single file for the entire simulation output, with a single axis for the focus point index. Since we stack delays and apodizations too, it makes sense to stack the simulation output into one xarray dataset